ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot

```diff
@@ -56,6 +57,43 @@ Run TestMnistPyTorchMCAD ODH test
     ...     CodeflareOperator
     Run Codeflare ODH Test    TestMnistPyTorchMCAD
 
+Run TestInstascaleMachinePool test
+    [Documentation]    Run Go E2E test TestInstascaleMachinePool
+    ...                This particular test runs on OSD cluster
+    [Tags]  Instascale
+    ...     Tier3
+    ...     DistributedWorkloads
+    ...     CodeflareOperator
+    Skip If RHODS Is Self-Managed
+
+    # Generate ocm token and create a secret
+    Log To Console    "Generating token ....."
+    ${token} =    Run Process    ocm token --generate
```

Comment:
This step expects user to be logged into OCM.
Is logging into OCM implicit for this test or does it have to be explicitly invoked?

Reply:
OCM login is implicit